docs/prd.md

```
@@ -0,0 +1,1093 @@
+
+## Tabla de contenidos
+
+1. [Descripción del software](#1-descripción-del-software)
+2. [Funciones principales](#2-funciones-principales)
+3. [Lean Canvas](#3-lean-canvas)
+4. [Casos de uso](#4-casos-de-uso)
+5. [Modelo de datos](#5-modelo-de-datos)
+6. [API (Alto-Nivel)](#6-api-alto-nivel)
+7. [Diseño de sistema a alto nivel](#7-diseño-de-sistema-a-alto-nivel)
+8. [Diagramas C4](#8-diagramas-c4)
+9. [Out of Scope / Non-Goals](#9-non-goals)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)